.github/workflows/application.yml

```diff
@@ -109,6 +109,60 @@ jobs:
                 run: vendor/bin/phpstan analyse
                 if: always() && steps.end-of-setup.outcome == 'success'
 
+    classes-specifictions:
+        runs-on: ubuntu-latest
+
+        name: "Classes specifications (PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }})"
+        strategy:
+            fail-fast: false
+            matrix:
+                php: ["7.4", "8.0"]
```

Comment:
```suggestion
                php: ["8.0"]
```

Reply:
I thought  you pushed it to master, on `1.10` I'm not sure how to make this PR green :D

Reply:
Nope, as we still support PHP 7.4 in 1.10.* versions :dancer: